pygeos/test/test_constructive.py

```diff
@@ -75,3 +75,25 @@ def test_snap_none():
 def test_snap_nan_float(geometry):
     actual = pygeos.snap(geometry, point, tolerance=np.nan)
     assert actual is None
+
+
+@pytest.mark.skipif(pygeos.geos_version < (3, 8, 0), reason="GEOS < 3.8")
+def test_make_valid_none():
+    actual = pygeos.make_valid(None)
+    assert actual is None
+
+
+@pytest.mark.skipif(pygeos.geos_version < (3, 8, 0), reason="GEOS < 3.8")
+@pytest.mark.parametrize(
+    "geom,expected",
+    [
+        (pygeos.points(0, 0), pygeos.points(0, 0)),
+        (
+            Geometry("POLYGON((0 0, 1 1, 1 2, 1 1, 0 0))"),
+            Geometry("MULTILINESTRING ((0 0, 1 1), (1 1, 1 2))"),
+        ),
+    ],
+)
+def test_make_valid(geom, expected):
+    actual = pygeos.make_valid(geom)
+    assert actual == expected
```

Comment:
Using numpy `assert_array_equal` here would let you handle 0d and 1d test cases above:

```
assert_array_equal(actual, expected)
```